### DISCUSSION TITLE
Add "peek at desktop" to the Windows 11 tasbkar style

### DISCUSSION THREAD

**Discussion by xMcSqueezy** [valinet/ExplorerPatcher#2234] [Ideas]
In Windows 10, there's an option to preview desktop just by hovering on the "show desktop" button without pressing it. I personally use that feature a lot, and that's one of the main reasons i use ExplorerPatcher in the first place, but recently I started to like the Windows 11 taskbar, and seeing that feature missing makes me a bit upset. I don't know how many people used it back in Windows 10, but I certainly did, and I'd love to see that being implemented to the Windows 11 taskbar style in this program

_Concept art_ of what it could look like:
![image](https://github.com/valinet/ExplorerPatcher/assets/102478229/7b26da91-889b-4a38-b7d5-bbe258b7e082)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by xMcSqueezy** [reply]
Windows 11 22H2 (10.0.22621.2361)
ExplorerPatcher Version 22621.2361.68.1

Here are my exported settings. Had to change the file type from .reg to .txt because github doesn't support uploading those though.
[ExplorerPatcher_22621.2361.58.1.txt](https://github.com/valinet/ExplorerPatcher/files/12801480/ExplorerPatcher_22621.2361.58.1.txt)

**Reply by pyrates999** [reply]
do you mean ExplorerPatcher Version 22621.2361.58.1?

**Reply by xMcSqueezy** [reply]
Yes, it's 58.1, I made a typo. The version in the .txt file is correct

**Reply by pyrates999** [reply]
Try this: https://www.minitool.com/news/how-to-enable-or-disable-peek-at-desktop.html

**Reply by xMcSqueezy** [reply]
Unfortunately, that didn't work for me. "Enable peek" was enabled by default and it didn't make a difference. My point was to introduce that feature into ExplorerPatcher, since it doesn't exist for the Windows 11 taskbar style, while it does on Windows 10.
![image](https://github.com/valinet/ExplorerPatcher/assets/102478229/7b26da91-889b-4a38-b7d5-bbe258b7e082)
This is what I mean, to have this pop-up while using the Windows 11 taskbar style. Currently, right-clicking on the bottom right corner does nothing. I just thought that'd be a neat idea.

**Reply by pyrates999** [reply]
Is this what windows 11 shows you when you right click on the desktop button on the right side?

**Reply by xMcSqueezy** [reply]
No, this is just concept art. I pasted the menu from Windows 10 onto a Windows 11 taskbar to just show how it would look like.

**Reply by pyrates999** [reply]
ah ok.
